test/unit/utility/views/chunk_test.cpp

```diff
@@ -0,0 +1,163 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
```

Comment:
Shouldn't this be 2022?

Reply:
yes. This has to be done in every file :see_no_evil: But i will correct it here already